GOVERNANCE.md

```diff
@@ -0,0 +1,82 @@
+# Governance
+
+This project uses [Martha's Rules][marthas-rules] for making decisions.
+The first section below describes these rules;
+the second lists current voting members of the project and explains how to become one,
+while the third lists decisions made to date.
+
+## Making Decisions
+
+1.  Before each voting deadline (discussed below),
+    anyone who wishes may sponsor a proposal by filing an issue in the GitHub repository tagged "governance"
```

Comment:
I assume " anyone who wishes" includes non-voting members, but I wanted to check to make sure.

Reply:
good catch - it's intended to be voting members only. (if a non-voting member wants to put forward a proposal, they need to find a voting member as a sponsor - this rule prevents spam.)